app/views/index.scala.html

```diff
@@ -0,0 +1,18 @@
+@()
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Rights statements</title>
+
+  </head>
+  <body>
+
+
+    <h2>Europeana &mdash; Rights statements </h2>
```

Comment:
Nitpicking, but it is not Europeana Rights Statements. Europeana is partner in a consortium, the statements are not solely from Europeana.